tools/downloader/converter.py

```diff
@@ -75,20 +60,20 @@ def cancel(self):
         self._future.cancel()
 
 
-def convert_to_onnx(context, model, output_dir, args):
-    context.printf('========= {}Converting {} to ONNX',
-                   '(DRY RUN) ' if args.dry_run else '', model.name)
+def convert_to_onnx(reporter, model, output_dir, args):
+    reporter.print_section_heading('{}Converting {} to ONNX',
+        '(DRY RUN) ' if args.dry_run else '', model.name)
 
     conversion_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
                                                                dl_dir=args.download_dir / model.subdirectory)
                                for arg in model.conversion_to_onnx_args]
     cmd = [str(args.python), str(Path(__file__).absolute().parent / model.converter_to_onnx), *conversion_to_onnx_args]
 
-    context.printf('Conversion to ONNX command: {}', common.command_string(cmd))
-    context.printf('')
+    reporter.print('Conversion to ONNX command: {}', common.command_string(cmd))
+    reporter.print(flush=True)
```

Comment:
Will it be equal to `reporter.print('Conversion to ONNX command: {}', common.command_string(cmd), flush=True)`?

Reply:
No, the second `print` is to print a blank line before the output of the command.